app/spiffworkflow/conditions/propertiesPanel/ConditionsPropertiesProvider.js

```
@@ -1,6 +1,7 @@
 import { is } from 'bpmn-js/lib/util/ModelUtil';
 import {
-  TextAreaEntry
+  TextAreaEntry,
+  isTextAreaEntryEdited
```

ESLint

[error] Insert `,`

(prettier/prettier)